example/spec/parallel/pythonReferenceTests/PythonReferenceSpec.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const workflowName = 'PythonReferenceWorkflow';
+
+const { ActivityStep, LambdaStep } = require('@cumulus/integration-tests/sfnStep');
+const { buildAndExecuteWorkflow } = require('@cumulus/integration-tests');
+
+const { loadConfig } = require('../../helpers/testUtils');
+
+describe('The Python Reference workflow', () => {
+  let config;
+  let lambdaStep;
+  let activityStep;
+  let workflowExecution;
+
+  const initialPayload = {
+    initialData: {
+      key1: 'value1',
+      key2: 'value2'
+    }
+  };
+
+  beforeAll(async () => {
+    config = await loadConfig();
+    process.env.ExecutionsTable = `${config.stackName}-ExecutionsTable`;
+
+    workflowExecution = await buildAndExecuteWorkflow(
+      config.stackName,
+      config.bucket,
+      workflowName,
+      null,
+      null,
+      initialPayload
+    );
+    lambdaStep = new LambdaStep();
+    activityStep = new ActivityStep();
+  });
+
+  it('executes successfully', () => {
+    expect(workflowExecution.status).toEqual('SUCCEEDED');
+  });
+
+  it('contains the expected output from the PythonReferenceTask', async () => {
+    const stepOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'PythonReferenceTask');
+    const expectedPayload = {
+      inputData: initialPayload.initialData,
+      configInputData: { key1: 'injectedData' },
+      newData: { newKey1: 'newData1' }
+    };
+
+    expect(stepOutput.payload).toEqual(expectedPayload);
+    expect(Object.keys(stepOutput)).toContain('meta');
+    expect(Object.keys(stepOutput)).toContain('cumulus_meta');
+  });
+
+  it('contains the expected output from the Process Task Activity', async () => {
+    const stepOutput = await activityStep.getStepOutput(workflowExecution.executionArn, 'EcsTaskPythonProcess');
+    const expectedPayload = {
+      fake_output1: 'first fake output',
+      fake_output2: 'second fake output'
+    };
+    expect(stepOutput.payload).toEqual(expectedPayload);
+    expect(Object.keys(stepOutput)).toContain('meta');
```

Comment:
Is this how we're proving integration with the CMA: by showing that the output from the activity was put in `payload` and that the `meta` and `cumulus_meta` keys still exist? 

I think this is a dumb question but why the different approach from integration testing the task where we are ensuring that task config is handled properly? Can activities not handle task config (he says realizing this is probably the case)?

Reply:
It's not a dumb question - at one point the two tasks were being built with identical requirements, and given how process-py utilizes the CMA, similar build envs would make that testing somewhat redundant.      That's no longer true 🤔